In [1]:
import pandas as pd
import numpy as np
import warnings
warnings.filterwarnings('ignore')

In [2]:
df = pd.read_csv('Q9_a.csv')
df.head()

Index  speed  dist
0      1      4     2
1      2      4    10
2      3      7     4
3      4      7    22
4      5      8    16

In [3]:
df.speed.skew()

-0.11750986144663393

In [4]:
df.dist.skew()

0.8068949601674215

In [5]:
df.skew()

Index    0.000000
speed   -0.117510
dist     0.806895
dtype: float64

# skewness value for speed and distance is -0.117510 and 0.806895

In [6]:
df.speed.kurt()

-0.5089944204057617

In [7]:
df.dist.kurt()

0.4050525816795765

In [8]:
df.kurt()

Index   -1.200000
speed   -0.508994
dist     0.405053
dtype: float64

# Kurtosis value for speed and distance is -0.508994 and 0.405053